File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/plagiarized/L4/09/Level4.py
import math

def main():
    radius, height = map(float, input("Enter the radius and length of a cylinder: ").split())
    
    area = calculate_base_area(radius)
    volume = calculate_volume(area, height)
    
    print(f"The area is {area:.5f}")
    print(f"The volume of the cylinder is {volume:.5f}")

def calculate_base_area(rad):
    return rad ** 2 * math.pi

def calculate_volume(base_area, height):
    return base_area * height

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/plagiarized/L4/09/Level4.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function hitungLuasAlas(rad: number): number {
  const luasAlas = rad * rad * 3.14159;
  return luasAlas;
}

function hitungVolume(luasAlas: number, tinggi: number): number {
  const volume = luasAlas * tinggi;
  return volume;
}

rl.question('Enter the radius and length of a cylinder: ', (input) => {
  const [radius, tinggi] = input.split(' ').map(Number);

  console.log('The area is', hitungLuasAlas(radius));
  console.log('The volume of the cylinder is', hitungVolume(hitungLuasAlas(radius), tinggi));

  rl.close();
});

